testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/IfMandrelVersion.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021, Red Hat Inc. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.graalvm.tests.integration.utils.versions;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Examples:
+ *
+ *     IfMandrelVersion(min = "20.1", max="20.3.2")
```

Comment:
It would be nice to document that this annotation would be qual to `20.1 <= GRAALVM_VERSION <= 20.3.2`. Particularly the matching is inclusive.

Reply:
@jerboaa THX. Will do.